python/OneSampleTTest.qmd

```diff
@@ -0,0 +1,66 @@
+---
+title: "One Sample t-test"
+---
+
+---
+subtitle: "One Sample t-test in Python"
+---

```

Comment:
The part outside of the code blocks just follow traditional markdown notation so like the following 
```suggestion
## One Sample t-test in Python
```